### PR TITLE
Styles and routing

### PR DESCRIPTION
**Pull request by jd487**

### CODE REVIEW

File: src/shared/ components/Nabar.tsx
Comment (dan-holmes) [inline src/shared/ components/Nabar.tsx:1]: Typo in file name? Also there's a rouge  space in the folder name

File: src/App.tsx [hunk 1]
```diff
@@ -1,26 +1,15 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Routes>
+      <Route path="/" element={<Navbar />}>
```
Comment (dan-holmes) [inline src/App.tsx]: Do we want the Navbar on every page? In which case would it sit outside of `<Routes>`?

**Comment by dan-holmes** [comment]
Looks good apart from those two comments. Happy for you to merge

File: src/App.tsx (hunk 1 above)
Comment (jd487) [reply src/App.tsx]: Responded to comments and merged @dan-holmes

**Pull request merged**